shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -117,15 +191,56 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   result(handled ? nil : FlutterMethodNotImplemented);
 }
 
+/**
+ * Updates the text input model with state received from the framework via the
+ * TextInput.setEditingState message.
+ */
+- (void)setEditingState:(NSDictionary*)state {
```

Comment:
Optional nit: you're free to change this going forward obviously, but I've always enforced that the style for the macOS embedding is to declare all private methods in the private class declaration and document them there. (I've always strongly preferred that style since it means the API surface and its documentation are grouped in two places (one for public, one for private).

Reply:
Done. Much better. Thanks.